docs/operations/metrics.md

```diff
@@ -516,8 +516,12 @@ These metrics are available on operating systems with the cgroup kernel feature.
 |------|-----------|----------|------------|
 |`cgroup/cpu/shares`|Relative value of CPU time available to this process. Read from `cpu.shares`.||Varies|
 |`cgroup/cpu/cores_quota`|Number of cores available to this process. Derived from `cpu.cfs_quota_us`/`cpu.cfs_period_us`.||Varies. A value of -1 indicates there is no explicit quota set.|
+|`cgroup/cpu/usage/percentage`|Total cpu percentage used by cgroup of process that is running||0-100|
```

Comment:
Is it possible to split this up by type of CPU usage? (user, sys, iowait, etc)?

Reply:
Only user and sys are exposed by the cgroups. 

Also, calculation results might be slightly different because user/sys time are provided in units of "jiffies" where as the total is provided as nanosecs.

Reply:
Modified.